src/core/server/logging/__snapshots__/logging_system.test.ts.snap

```diff
@@ -1,20 +1,20 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`appends records via multiple appenders.: console logs 1`] = `"[2012-01-31T23:33:22.011Z][INFO ][some-context] You know, just for your info."`;
+exports[`appends records via multiple appenders.: console logs 1`] = `"[2012-01-31T18:33:22.011-05:00][INFO ][some-context] You know, just for your info."`;
 
 exports[`appends records via multiple appenders.: file logs 1`] = `
-"[2012-01-31T23:33:22.011Z][WARN ][tests] Config is not ready!
+"[2012-01-31T13:33:22.011-05:00][WARN ][tests] Config is not ready!
 "
 `;
 
 exports[`appends records via multiple appenders.: file logs 2`] = `
-"[2012-01-31T23:33:22.011Z][ERROR][tests.child] Too bad that config is not ready :/
+"[2012-01-31T08:33:22.011-05:00][ERROR][tests.child] Too bad that config is not ready :/
 "
 `;
 
 exports[`asLoggerFactory() only allows to create new loggers. 1`] = `
 Object {
-  "@timestamp": "2012-01-31T18:33:22.011-05:00",
+  "@timestamp": "2012-01-30T22:33:22.011-05:00",
```

Comment:
The updates to other snapshots make sense - but it's interesting that these specific snapshots with `@timestamp` changed too. I see we are mocking the timestamp in the tests, but perhaps we should be mocking the default TZ as well? (I might be missing something here)

Reply:
@lukeelmers we're using the [global timezone mock](https://github.com/elastic/kibana/blob/master/packages/kbn-test/src/jest/setup/mocks.js#L26-L35) for `moment-timezone` that mocks the `tz` as `America/New_York`, hence the consistent offset of -5 hrs. I'm still trying to figure out why the actual date/time changed.